### PR TITLE
Added new divider tokens for desktop and mobile

### PR DESCRIPTION
**Pull request by KayiuCarlson**
## Description

Added new divider tokens for both desktop and mobile.

## Motivation and context

Added tokens for horizontal and vertical orientation.

## Related issue

N/A

## Types of changes

- [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue)
- [x] Minor (add a new token, changing a value; non-breaking change which adds functionality)
- [ ] Major (deleting a token, changing token value type; fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.)

### CODE REVIEW

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/7703114484)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.divider-horizontal-minimum-width
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.divider-horizontal-minimum-width
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.divider-vertical-minimum-height
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.divider-vertical-minimum-height
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (4) divider-horizontal-minimum-width/desktop, divider-horizontal-minimum-width/mobile, divider-vertical-minimum-height/desktop, divider-vertical-minimum-height/mobile
CHANGED VALUES: (7) field-label-top-margin-extra-large/desktop, field-label-top-margin-large/desktop, field-label-top-margin-medium/desktop, field-label-top-to-asterisk-extra-large/desktop, field-label-top-margin-extra-large/mobile, field-label-top-margin-large/mobile, field-label-top-margin-medium/mobile impacting 4 tokens
CHANGED SCHEMAS: (1) corner-radius-1000
```
</p></details>

TEST COMPLETE

**Comment by mrcjhicks** [comment]
If the intent is to have the same values for desktop & mobile, we can probably set that "constant-token-duplicate" flag in the metadata to "true".

**Comment by GarthDB** [comment]
@mrcjhicks, we've had a few tokens that have come through with the same value for desktop and mobile. Could we update all of them? We would just need to make sure the UUIDs don't get messed up.

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json [hunk 1]
```diff
@@ -5059,6 +5059,28 @@
...
     }
   },
+  "divider-horizontal-minimum-width": {
+    "value": "200px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "divider-horizontal-minimum-width",
+        "constant-token-duplicate": false
```
Comment (GarthDB) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json:5069]: Could this be set to `true` if they are the same value for desktop and mobile?

Comment (mrcjhicks) [reply src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json:5069]: https://github.com/adobe/spectrum-tokens-studio-data/pull/47#issuecomment-1915676370

**Comment by mrcjhicks** [comment]
> @mrcjhicks, we've had a few tokens that have come through with the same value for desktop and mobile. Could we update all of them? We would just need to make sure the UUIDs don't get messed up.

If we don't want this feature we should just remove it. I think we discussed this last week or the week before. I don't remember if there was as decision, though.

**Comment by lynnhao** [comment]
> If the intent is to have the same values for desktop & mobile, we can probably set that "constant-token-duplicate" flag in the metadata to "true".

Hi that's my fault! The intent is to have that as "true" but I told Kayiu she could leave it, since I had thought that would be updated programmatically. Let me know if we should be doing that manually instead. cc @karstens

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/7706075732)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.divider-horizontal-minimum-width
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.divider-vertical-minimum-height
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (2) divider-horizontal-minimum-width, divider-vertical-minimum-height
```
</p></details>

TEST COMPLETE

**Comment by mrcjhicks** [comment]
**[SYNCING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/7706083333)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.divider-horizontal-minimum-width
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.divider-vertical-minimum-height
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (2) divider-horizontal-minimum-width, divider-vertical-minimum-height
```
</p></details>

WROTE: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json`
WROTE: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json`
WROTE: `/spectrum-tokens/packages/tokens/src/layout-component.json`

SYNC COMPLETE

[Spectrum Tokens Pull Request](https://github.com/adobe/spectrum-tokens/pull/253)
[Tokens Studio Pull Request](https://github.com/adobe/spectrum-tokens-studio-data/pull/49)

**Comment by karstens** [comment]
@GarthDB @mrcjhicks @lynnhao I see two conflicting use-cases for the `constant-token-duplicate` flag.
1. It was introduced as a helper, to check via the diff-script, if the token values across set _do_ match. And log a warning for the token admins to double-check.
2. It is (or was) used by the diff script to create a constant token (without nested sets) if the values match and the `constant-token-duplicate` flag is true.

I would vote for removing that `constant-token-duplicate` flag, because the diff script is smart enough to compare the token values directly and create a the constant token without the need of the `constant-token-duplicate` flag.

**Pull request merged**